### PR TITLE
DOCSP-37612: v6.5.0 release

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-37612>
Staging:
- [Compatibility](https://preview-mongodbcchomongodb.gatsbyjs.io/node/DOCSP-37612-v6.5.0/compatibility/#compatibility-table-legend)
- [What's New](https://preview-mongodbcchomongodb.gatsbyjs.io/node/DOCSP-37612-v6.5.0/whats-new/)
- [New version displayed](https://preview-mongodbcchomongodb.gatsbyjs.io/node/DOCSP-37612-v6.5.0/quick-start/download-and-install/#install-the-node.js-driver)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -56,6 +57,44 @@ Learn what's new in:
...
 
+.. _version-6.5:
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moved ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
```
Comment (rustagir) [inline source/whats-new.txt:68]: S: match tense
```suggestion
- Moves ``ObjectId`` generation to the ``pkFactory`` class for bulk write
  operations.
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -56,6 +57,44 @@ Learn what's new in:
...
+
+.. warning::
+
+   If you previously specified the ``pkFactory`` to handle bulk writes,
+   the ``_id`` fields of the documents inserted by using bulk writes may
+   be inconsistent with the behavior in this version.
```
Comment (rustagir) [inline source/whats-new.txt:74]:
```suggestion
   If you previously specified that a ``pkFactory`` instance handles bulk writes,
   the ``_id`` fields of the documents inserted by using bulk writes may
   be inconsistent with the behavior in this version.
```

File: source/whats-new.txt [hunk 3]
```diff
@@ -56,6 +57,44 @@ Learn what's new in:
...
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moved ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
+
+.. warning::
```
Comment (rustagir) [inline source/whats-new.txt:70]: Q: should this warning be indented to be under the preceding bullet point?

File: source/whats-new.txt [hunk 4]
```diff
@@ -56,6 +57,44 @@ Learn what's new in:
...
+
+   If you previously specified the ``pkFactory`` to handle bulk writes,
+   the ``_id`` fields of the documents inserted by using bulk writes may
+   be inconsistent with the behavior in this version.
+
+- Fixes the read preference sent with read operations when directly connected
+  to a secondary in the replica set to ``primaryPreferred``.
```
Comment (rustagir) [inline source/whats-new.txt:77]:
```suggestion
- Sets the read preference that is sent with read operations to ``primaryPreferred`` when
  you are directly connected to a secondary in the replica set.
```

File: source/whats-new.txt [hunk 5]
```diff
@@ -56,6 +57,44 @@ Learn what's new in:
...
+
+- Adds the original error document returned by the server to the
+  ``errorResponse`` field of the ``MongoServerError`` document.
+
+- Deprecates the ``CloseOptions`` interface which is unused by the driver.
+
+To learn more about this release, see the
+`v6.5.0 Release Highlights
+<https://github.com/mongodb/node-mongodb-native/releases/tag/v6.5.0>`__ on
+GitHub.
```
Comment (rustagir) [inline source/whats-new.txt:96]: S: maybe out of scope, but I noticed the v6.4 section is missing this link

**Review by rustagir — APPROVED** [review]
A few small things, but I'll take another look if you want!

File: source/whats-new.txt (hunk 3 above)
Comment (ccho-mongodb) [reply source/whats-new.txt:70]: I'm not sure you can put an admonition in a list, but I'll try it out.

File: source/whats-new.txt (hunk 2 above)
Comment (ccho-mongodb) [reply source/whats-new.txt:74]: Good call! Will update to clarify an instance.

File: source/whats-new.txt (hunk 4 above)
Comment (ccho-mongodb) [reply source/whats-new.txt:77]: Thanks for the rewording. I'll use parts of this suggestion, but will continue to use the verb "fixes" to make sure it's understood that way. From the release notes:

> Fixed applying read preference to commands depending on topology

File: source/whats-new.txt (hunk 3 above)
Comment (ccho-mongodb) [reply source/whats-new.txt:70]: Unfortunately, that didn't work (unexpected indentation error)

File: source/whats-new.txt [hunk 6]
```diff
@@ -56,6 +57,45 @@ Learn what's new in:
...
 
+.. _version-6.5:
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moves ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
```
Comment (baileympearson) [inline source/whats-new.txt:68]:
```suggestion
- Bulk write operations generate document ids using the ``pkFactory``
```

**Review by baileympearson — CHANGES_REQUESTED** [review]
one wording change, otherwise looks good

File: source/whats-new.txt (hunk 6 above)
Comment (ccho-mongodb) [reply source/whats-new.txt:68]: Thanks, I'll change "ObjectID generation" to "generate document ids".
To maintain the style, the bullet point needs to start with a present tense verb. I'll change it to the following (feel free to suggest something else):

"Updates bulk write operations to use the pkFactory class for document id generation."

Comment (baileympearson) [reply source/whats-new.txt]: yup, that works too.  thanks!

**Review by baileympearson — APPROVED** [review]

**Pull request merged**